lib/github-model.js

```
@@ -0,0 +1,82 @@
+      };
+    try {
+      const parsedYaml = yaml.load(yamlContent);
+      return {
+        urgency: parsedYaml.urgency || null,
+        importance: parsedYaml.importance || null
+      };
+    } catch (parseError) {
+      console.error('Failed to parse YAML response:', parseError.message);
+      console.error('Raw response:', rawContent);
+      throw new Error('Failed to parse model response');
+    }
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement will throw an exception ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)